bench_runner/templates/_benchmark.src.yml

```diff
@@ -112,7 +114,9 @@ jobs:
           python-version: "3.11"
       - name: Building Python and running pyperformance
         run: |
-          python workflow_bootstrap.py ${{ inputs.fork }} ${{ inputs.ref }} ${{ inputs.machine }} ${{ inputs.benchmarks || 'all' }} ${{ env.flags }} ${{ inputs.force && '--force' || '' }} ${{ inputs.pgo && '--pgo' || '' }} ${{ inputs.perf && '--perf' || '' }} --run_id ${{ github.run_id }}
+          python workflow_bootstrap.py ${{ inputs.fork }} ${{ inputs.ref }} \
+              ${{ (inputs.machine == 'all' || inputs.machine == '__really_all') && inputs.machine || '"$BENCHMARK_RUNNER_NAME"' }} \
```

Comment:
I think this can just be `$BENCHMARK_RUNNER_NAME`, and the special logic in `workflow.should_run` to handle `all` and `__really_all` could be removed.  This is a bit of vestige from when the individual jobs were more entwined.